Resources/Private/Partials/List/ListItem.html

```diff
@@ -2,12 +2,70 @@
       xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
 >
 
-<f:variable name="targetPlugin" value="{f:if(condition: '{settings.detailPid} > 0 && {settings.detailPid} != {data.pid}', then: 'Detail', else: 'ListAndDetail')}" />
+<f:variable name="targetPlugin" value="{f:if(condition: '{settings.detailPid} > 0 && {settings.detailPid} != {data.pid}', then: 'Detail', else: 'ListAndDetail')}"/>
 
 <li>
-    <f:link.action pageUid="{settings.detailPid}" action="detail" pluginName="{targetPlugin}" arguments="{profile: profile}">
+    <f:link.action
+        pageUid="{settings.detailPid}"
+        action="detail"
+        pluginName="{targetPlugin}"
+        arguments="{profile: profile}"
+    >
         {profile.firstName}{f:if(condition: profile.middleName, then: ' {profile.middleName}')} {profile.lastName}
     </f:link.action>
+    <br>
+    <f:if condition="{profile.contracts}">
+        <f:for each="{profile.contracts}" as="contract">
+            <f:render section="Contract" arguments="{contract: contract, settings: settings}" />
+        </f:for>
+    </f:if>
 </li>
 
+<f:section name="Contract">
+    <f:if condition="{settings.showFields}" >
```

Comment:
What happens if no fields are selected at all ? Display nothing ?

Empy field select === hide contracts .. do I get this right ?